env.j2

```diff
@@ -0,0 +1,11 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+TRELLO_KEY={{trello-key}}
+TRELLO_TOKEN={{trello-token}}
+TRELLO_BOARDID={{trello-boardid}}
+TRELLO_TODOID={{trello-todoid}}
+TRELLO_DONEID={{trello-doneid}}
```

Comment:
Remove the hyphens. Your variable names in your playbook.yml file don't have hyphens. E.g. `name: trellokey`. So the variable names here shouldn't either.

Also, it's not really a valid variable name. To use a variable called `trello-key` instead of calculating trello minus key, you have to do `vars['trello-key']`. Best to just drop the hyphen.